export const crashMainProcess = () => {
  throw new Error('oops')
}
